buildspec.yml

```diff
@@ -1,165 +1,10 @@
-          -var "spel_customreporpm7=${SPEL_CUSTOMREPORPM7}" \
-          -var "spel_customreponame7=${SPEL_CUSTOMREPONAME7}" \
-          -var "spel_disablefips=${SPEL_DISABLEFIPS}" \
-          -var "spel_epel7release=${SPEL_EPEL7RELEASE}" \
-          -var "spel_epelrepo=${SPEL_EPELREPO}" \
-          -var "spel_extrarpms=${SPEL_EXTRARPMS}" \
-          -var "spel_identifier=${SPEL_IDENTIFIER}" \
-          -var "spel_version=${SPEL_VERSION}" \
-          -var "source_ami_centos7_hvm=${SOURCE_AMI_CENTOS7_HVM}" \
-          -var "source_ami_rhel7_hvm=${SOURCE_AMI_RHEL7_HVM}" \
-          -var "ssh_interface=${SSH_INTERFACE}" \
-          -var "subnet_id=${SUBNET_ID}" \
-          -var "vpc_id=${VPC_ID}" \
-          spel/minimal-linux.json
-      - |
-        for BUILDER in $(echo $SPEL_BUILDERS | sed -e 's/,/ /g')
-        do
-          AMI_NAME="${SPEL_IDENTIFIER}-${BUILDER}-${SPEL_VERSION}.x86_64-gp2"
-          BUILDER_ENV=$(echo $BUILDER | sed -e 's/\./_/g' -e 's/-/_/g')
-          export $BUILDER_ENV=$(export AWS_DEFAULT_REGION=$AWS_REGION; aws ec2 describe-images --filters Name=name,Values="$AMI_NAME" --query 'Images[0].ImageId' --out text --profile $SPEL_IDENTIFIER)
-        done
-      - |
-        AWS_PROFILE=$SPEL_IDENTIFIER ./packer build \
-          -only "${SPEL_BUILDERS}" \
-          -var "aws_region=${AWS_REGION}" \
-          -var "pip_url=${PIP_URL}" \
-          -var "pypi_url=${PYPI_URL}" \
-          -var "spel_identifier=${SPEL_IDENTIFIER}" \
-          -var "spel_version=${SPEL_VERSION}" \
-          -var "spel_disablefips=${SPEL_DISABLEFIPS}" \
-          -var "ssh_interface=${SSH_INTERFACE}" \
-          -var "subnet_id=${SUBNET_ID}" \
-          -var "vpc_id=${VPC_ID}" \
-          tests/minimal-linux.json
-  post_build:
-    commands:
-      - echo "Sleeping 180 seconds to ensure the AMI Name is searchable..."; sleep 180
-      - |
-      - |
-        for BUILDER in $(echo $SPEL_BUILDERS | sed -e 's/,/ /g')
-        do
-          AMI_NAME="validation-${SPEL_IDENTIFIER}-${BUILDER}-${SPEL_VERSION}.x86_64-gp2"
-          AMI_ID=$(export AWS_DEFAULT_REGION=$AWS_REGION; aws ec2 describe-images --filters Name=name,Values="$AMI_NAME" --query 'Images[0].ImageId' --out text --profile $SPEL_IDENTIFIER)
-          (export AWS_DEFAULT_REGION=$AWS_REGION; test "$AMI_ID" != "None" && aws ec2 deregister-image --image-id $AMI_ID --profile $SPEL_IDENTIFIER)
-          if [ "${SPEL_CI}" = "true" ]
-          then
-            AMI_NAME="${SPEL_IDENTIFIER}-${BUILDER}-${SPEL_VERSION}.x86_64-gp2"
-            AMI_ID=$(export AWS_DEFAULT_REGION=$AWS_REGION; aws ec2 describe-images --filters Name=name,Values="$AMI_NAME" --query 'Images[0].ImageId' --out text --profile $SPEL_IDENTIFIER)
-            (export AWS_DEFAULT_REGION=$AWS_REGION; test "$AMI_ID" != "None" && aws ec2 deregister-image --image-id $AMI_ID --profile $SPEL_IDENTIFIER)
-          fi
-        done
-      - echo "Packer build completed on $(date)"
+      - make
```

Comment:
One important detail of the buildspec stages is that `post_build` runs even if `build` fails. This gives us a "cleanup" phase, which we were using to ensure that any AMIs that need to be removed are removed.

* https://docs.aws.amazon.com/codebuild/latest/userguide/view-build-details.html#view-build-details-phases

We still need that cleanup step here, and in .gitlab-ci.yml...